#3 done

Vaarun-Kamath opened the pull request.

Commit:
#3 done

PhoenixFlame101 commented:
Here's some changes that would fetch you the full bounty:

 - Make the game over text bigger and bold
 - Make the buttons a little shorter in width and vertically centered

Great job so far, keep up the good work.

Vaarun-Kamath commented:
@PhoenixFlame101 yes i will work on that now

Commit:
#3- Quit Screen (Better Looking)

Made it look better pls ignore the main menu I will fix that also and push in other branch since its another issue also

PhoenixFlame101 commented:
The game over screen looks good, but the main scene has been to the previous version of the game over screen(?). Fix this and the bounty points are yours!

Commit:
#2 and #3 solved

PhoenixFlame101 commented:
Can you commit with only the game over scene, and the main menu scene removed? Because otherwise they'll probably be merge conflicts.

Commit:
#3-Final

PhoenixFlame101 commented:
Looks good, but the score is not reset after a game restart. That seems to be the only issue right now.

Vaarun-Kamath commented:
okay working on it now

Commit:
#3-Fixed Score not resetting

PhoenixFlame101 commented:
Great job!

PhoenixFlame101 commented:
!bounty 80

PhoenixFlame101 commented:
!bounty 60

NavinShrinivas commented:
!bounty 40
